tensorflow/python/estimator/canned/boosted_trees_test.py

```diff
@@ -155,6 +161,12 @@ def before_run(self, run_context):
     self.assertAllClose([[0], [0], [0], [0], [0]],
                         [pred['class_ids'] for pred in predictions])
 
+    with self.assertRaisesRegexp(ValueError, 'empty'):
```

Comment:
lets move these to a separate test testFeatureImportancesOnEmtpyEnsemble.
U might also want to add a test on a ensemble that just has a root node (u expect it to raise only with normalize=true then)

Reply:
It seems that you already have such a test. So i would just remove here